_posts/en/newsletters/2023-02-22-newsletter.md

```diff
@@ -0,0 +1,212 @@
+    backup is to enter it into a trusted computing device and see if it
+    derives the expected public keys---but determining whether a device
+    should be trusted is often not a trivial procedure.  Worse, in order
+    to verify the integrity of existing SSSS shares (e.g. in SLIP39),
+    the user must bring together each share they want to verify with enough
+    other shares to reach the threshold and then enter them into a trusted
+    computing device.  That means verifying share integrity negates a
+    large benefit of having shares in the first place---the ability to
+    keep information safe and secure by distributing it across multiple
+    places or people.  With Codex32, users can verify the integrity of
+    each share individually on a regular basis using just paper, pen,
+    some printed materials, and a few minutes of time.
+
+    Discussion on the mailing list mainly focused on the differences
+    between Codex32 and SLIP39, which has been used in production for a
+    couple years now.  We recommend anyone interested in Codex32
+    investigate its [website][codex32 website] or watch a
+    [video][codex32 video] by one of its authors.  With the draft BIP,
+    the authors hope to see wallets begin to add support for using
+    Codex32-encoded seeds.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 23.02rc2][] is a release candidate for a new
+  maintenance version of this popular LN implementation.
+
+<!-- FIXME:harding
+
+- BDK, rusqlite vuln
+
+-->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #24149][] Signing support for Miniscript Descriptors FIXME:Xekyo
+
+- [Bitcoin Core #25344][] updates the `bumpfee` and `psbtbumpfee` RPCs
+  for using [Replace By Fee][topic rbf] (RBF) to allow specifying a
```

Comment:
Ah, I see what you mean, @Xekyo, without the "for" it would seem that those RPCs could use means _other than_ RBF to do something ... which is incorrect; they always / can only use RBF. I still think it's a little unclear; maybe something like
```suggestion
  , which use [Replace By Fee][topic rbf] (RBF), to allow the specification of a
```